spec/steps/start_page/integration_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe 'User can see an application status', type: :system do
-  let(:claim) { Claim.create(office_code:, solicitor:, firm_office:, claim_type:, rep_order_date:) }
-
+  let(:claim) { Claim.create(ufn:, office_code:, solicitor:, firm_office:, claim_type:, rep_order_date:) }
+  let(:ufn) { '120223/001' }
```

Comment:
why have we updated this? does the test fail without it? if so why?

Reply:
yes in the form def - it has a presence field it does fail without